fr-ln/jw300-baseline/baseline.ipynb

```diff
@@ -0,0 +1,6194 @@
+{
```

Comment:
<p>So, people were have issues with this on colab. Did you train locally?</p><p> </p><p>Maybe we can add comments to this affect (to this and to the starter_notebook):</p><p> </p><p><code>#If you're not running on colab, uncomment the lines below as they'll help your code run faster</code></p><p><code>&lt;your code here&gt;</code></p><p><code> </code></p><p><code>#If you are running on colab, the lines below will work - but they do take long for big datasets</code></p><p><code>&lt;put old code here&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>Okay...</p><p>That would be nice...</p><p>I used the code to train on GCP but not locally on my personal laptop.</p><p>I would be nice if you tell me which specific issues people are having on Colab.</p><p>I will definitely add those line both on Colab and this notebook...</p>

Reply:
Are people still having issues with this in the starter notebooks? 🤔
@jaderabbit ?